editor/src/clj/editor/cljfx_form_view.clj

```diff
@@ -178,7 +178,7 @@
 (defmethod form-input-view :default [{:keys [value type] :as field}]
   {:fx/type fx.label/lifecycle
    :wrap-text true
-   :text (str type " " (if (nil? value) "***NIL***" value) " " field)})
+   :text (str value)})
```

Comment:
Why was it like that before? Anyway, after the change, `type` and `field` are unused.

Reply:
Previously, the fallback view was shown if we used some new, undefined field type in `meta.edn`. It was only used for development. It started seeing it again when typing `foo.type=...` text in `game.properties` files. I think it's not seen anymore since I then added `type` validation in `settings-core` ns:
```clj
  (letfn [(parse-type [s]
            (case s
              nil :string
              "bool" :boolean
              "string_array" :list
              (let [k (keyword s)]
                (if (contains? (.getMethodTable ^MultiFn parse-setting-value) k)
                  k
                  :string))))
  ...)
```